src/ui/components/Passport/Passport.tsx

```diff
@@ -26,14 +24,14 @@ const Passport = () => {
   const showAddComment = shouldShowAddComment(nags);
   const showJumpToCode = shouldShowJumpToCode(nags);
   const showAddUnicornBadge = shouldShowAddUnicornBadge(nags);
-  const showRecordReplay = shouldShowRecordReplay(nags);
-  const showExploreSources = shouldShowExploreSources(nags);
   const showSearchSourceText = shouldShowSearchSourceText(nags);
-  const showLaunchCommandPalette = shouldShowLaunchCommandPalette(nags);
+  const showShareNag = shouldShowShareNag(nags);
   const showJumpToEvent = shouldShowJumpToEvent(nags);
   const showInspectElement = shouldShowInspectElement(nags);
   const showUseFocusMode = shouldShowUseFocusMode(nags);
-  const [stepIndex, setStepIndex] = useState(0);
+  const [selectedIndices, setSelectedIndices] = useState({ sectionIndex: 0, itemIndex: 0 });
+  const stepNames = ["step-one", "step-two", "step-three"] as const;
```

Comment:
:speech_balloon: My only other note is that lines 33-34 should probably be moved outside of the component on general principle, since they're static data that isn't changing. But that doesn't need to hold up this PR - feel free to move those in the next one.